src/common/options/global.yaml.in

```diff
@@ -6875,3 +6875,18 @@ options:
   level: dev
   default: 0
   desc: When EC writes should generate PDWs (development only) 0=optimal 1=never 2=when possible
+- name: daemon_uniquifier
+  type: str
+  level: advanced
+  desc: Unique string to be returned in PerfCounters
+  fmt_desc: A unique string to be selected at daemon
+    creation time and passed to the daemon at startup, for the use of
+    the dashboard and other monitoring.
```

Comment:
Who's enforcing this uniqueness? User? Cephadm? I don't see any check to guarantee that each service instance/daemon has a uniquely assigned ID.

Is this uniqueness enforced/expected per host, per service type, only RGW, or global. Can I unequivocally use this ID to locate any service regardless of their type?

Reply:
Repushed with more specific documentation.